etc/kubernetes/manifests/overlays/staging/strimzi/namePrefixConfig.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: builtin
+kind: PrefixSuffixTransformer
```

Comment:
What is this for?

Reply:
It adds a prefix to the Roles and RoleBindings. That's to make sure they won't conflict with the ones already present in the cluster.

Reply:
I see. Nice thing!